src/Kunstmaan/MediaBundle/Helper/MimeTypeGuesserFactory.php

```diff
@@ -12,12 +13,12 @@ class MimeTypeGuesserFactory implements MimeTypeGuesserFactoryInterface
      *
      * NOTE: If you override this, you'll probably still have to register the SVGMimeTypeGuesser as last guesser...
      *
-     * @return MimeTypeGuesser
+     * @return MimeTypesInterface
      */
-    public function get()
+    public function get(): MimeTypesInterface
```

Comment:
This is a BC break. I would also leave this class as is and deprecate it, so it can be replaced by the mime component. The component already has built-in support for SVG's, so no need for a custom guesser